### PR TITLE
Fix push to gcr

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- [x] Enables [use the env specified registry](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/3fa7ed650d218b687beb2cfe674a8067372dab4d)
- [x] [stop use of save-output(::) as required by GitHub](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/4a7b5ea3add51580ac8ca1ab223083e45cc92aa3)
- [x] [use v3 of GitHub checkout(node-16)](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/07ac9c381956b97bf250f371fd576958931aaa5d)
- [x] Trigger all workflows to run

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Details available at https://github.com/airqo-platform/AirQo-api/actions/runs/3599656814
- image
- image
- image
- image

**_HOW DO I TEST OUT THIS PR?_**
- Static code analysis

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

File: .github/workflows/deploy-apis-to-production.yml [hunk 1]
```diff
@@ -1518,7 +1518,7 @@ jobs:
       - name: Login to GCR
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.REGISTRY_URL }}
+          registry: ${{ env.REGISTRY_URL }}
           username: _json_key
           password: ${{ secrets.GCR_CONFIG }}
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:1523]: The first bug was caused when github actions was trying to reference a wrong variable. This address the first issue

File: .github/workflows/deploy-previews.yml [hunk 2]
```diff
@@ -289,7 +289,7 @@ jobs:
       url: ${{ steps.preview-url.outputs.url }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-previews.yml:293]: [use v3 of GitHub checkout(node-16)](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/07ac9c381956b97bf250f371fd576958931aaa5d)

File: .github/workflows/deploy-previews.yml [hunk 3]
```diff
@@ -648,7 +648,7 @@ jobs:
             --format='value(status.url)' \
             --platform managed \
             --region ${{ secrets.REGION }})
-          echo "::set-output name=url::${service_url}"
+          echo "url=${service_url}" >>$GITHUB_OUTPUT
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-previews.yml:652]: [stop use of save-output(::) as required by GitHub](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/4a7b5ea3add51580ac8ca1ab223083e45cc92aa3)

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

File: .github/workflows/deploy-apis-to-production.yml (hunk 1 above)
Comment (123MwanjeMike) [reply .github/workflows/deploy-apis-to-production.yml:1523]: [use the env specified registry](https://github.com/airqo-platform/AirQo-api/pull/1213/commits/3fa7ed650d218b687beb2cfe674a8067372dab4d)

**Review by NoahNsimbe — APPROVED** [review]
LGTM... thanks @123MwanjeMike

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**